<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>821</width>
      <height>571</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/images/HeartWaveDevice(v3).PNG</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="leftButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>380</y>
      <width>31</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
image:url(:/new/prefix1/images/buttons/leftButton.png);
border-radius: 20px;}
QPushButton:pressed {
	
	background-color: rgb(252, 233, 79);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="upButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>360</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	
image: url(:/new/prefix1/images/buttons/upButton.png);
border-radius: 20px;}
QPushButton:pressed {
background-color: #ffd300;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="downButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>430</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
image: url(:/new/prefix1/images/buttons/downButton.png);
border-radius: 20px;
}
QPushButton:pressed {
	background-color: rgb(252, 233, 79);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="rightButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>380</y>
      <width>21</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
image: url(:/new/prefix1/images/buttons/rightButton.png);
border-radius: 20px;}

QPushButton:pressed {
	background-color: rgb(252, 233, 79);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="selectButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>390</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-image: url(:/new/prefix1/images/buttons/selectButton.png);
border-image: url(:/new/prefix1/images/buttons/selectButton.png);
image: url(:/new/prefix1/images/buttons/selectButton.png);
border-radius: 20px;
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 3, y1: 3, x2: 3, y2: 3,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="powerButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>370</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
image: url(:/new/prefix1/images/buttons/powerButton.png);
border-radius: 20px;}

QPushButton:pressed {
	background-color: qlineargradient(x1: 3, y1: 3, x2: 3, y2: 3,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>370</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(94, 94, 94);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="backButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background-image: url(:/new/prefix1/images/buttons/arrow-undo-up-left-svgrepo-com.svg);
border-image: url(:/new/prefix1/images/buttons/arrow-undo-up-left-svgrepo-com.svg);
image: url(:/new/prefix1/images/buttons/arrow-undo-up-left-svgrepo-com.svg);
	background-color: rgb(94, 94, 94);
border-radius: 20px;
}
QPushButton:pressed {
	background-color: #ffd300;
}

</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="Battery">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/images/buttons/battery-twotone-70-svgrepo-com.svg);
border-image: url(:/new/prefix1/images/buttons/battery-twotone-70-svgrepo-com.svg);
image: url(:/new/prefix1/images/buttons/battery-twotone-70-svgrepo-com.svg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="HRContact">
    <property name="geometry">
     <rect>
      <x>763</x>
      <y>60</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
image: url(:/new/prefix1/images/buttons/heart-rate-1-svgrepo-com.svg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="MenuButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>470</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
background-color: rgb(36, 90, 171);
color: rgb(255, 255, 255);
border-radius: 20px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
    </property>
    <property name="text">
     <string>MENU</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>519</x>
      <y>109</y>
      <width>271</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <widget class="QLabel" name="AchievementLabel">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>0</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Achievement</string>
     </property>
    </widget>
    <widget class="QLabel" name="Achievemen_Int_Label">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
    <widget class="QLabel" name="LengthLabel">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>0</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Length</string>
     </property>
    </widget>
    <widget class="QLabel" name="length_Int_Label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>00.00</string>
     </property>
    </widget>
    <widget class="QLabel" name="coherencLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Coherence</string>
     </property>
    </widget>
    <widget class="QLabel" name="Coherence_Int_Label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>31</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="blocker">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>100</y>
      <width>291</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="menuList">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>100</y>
      <width>291</width>
      <height>171</height>
     </rect>
    </property>
    <property name="currentRow">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QListWidget" name="summaryView">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>108</y>
      <width>275</width>
      <height>50</height>
     </rect>
    </property>
    <property name="currentRow">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="RGB_Indicator">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>70</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(138, 226, 52);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="historyList">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>100</y>
      <width>291</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="historyDataView">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>100</y>
      <width>291</width>
      <height>85</height>
     </rect>
    </property>
    <property name="currentRow">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QListWidget" name="settingsList">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>100</y>
      <width>291</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>505</x>
      <y>280</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="breathPacer">
    <property name="geometry">
     <rect>
      <x>505</x>
      <y>280</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(138, 226, 52);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="batteryDeplete">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>580</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Deplete Battery</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signalDC">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>580</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Signal Disconnect</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>leftButton</zorder>
   <zorder>upButton</zorder>
   <zorder>downButton</zorder>
   <zorder>rightButton</zorder>
   <zorder>selectButton</zorder>
   <zorder>powerButton</zorder>
   <zorder>frame_2</zorder>
   <zorder>Battery</zorder>
   <zorder>HRContact</zorder>
   <zorder>MenuButton</zorder>
   <zorder>customPlot</zorder>
   <zorder>menuList</zorder>
   <zorder>summaryView</zorder>
   <zorder>RGB_Indicator</zorder>
   <zorder>historyList</zorder>
   <zorder>historyDataView</zorder>
   <zorder>settingsList</zorder>
   <zorder>blocker</zorder>
   <zorder>label_2</zorder>
   <zorder>breathPacer</zorder>
   <zorder>batteryDeplete</zorder>
   <zorder>signalDC</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
